test/fixtures/mysql-private/main.tf

```
@@ -15,8 +15,9 @@
  */
 
 module "example" {
-  source       = "../../../examples/mysql-private"
-  project_id   = var.project_id
-  network_name = var.network_name
+  source            = "../../../examples/mysql-private"
+  project_id        = var.project_id
+  network_name      = var.network_name
+  cloudsql_mysql_sa = var.cloudsql_mysql_sa
```

```suggestion
  cloudsql_mysql_sa = google_service_account.cloudsql_mysql_sa.email
```
This uses the service account created in the test fixture:
```terraform
resource "google_service_account" "cloudsql_mysql_sa" {
  project    = module.project.project_id
  account_id = "cloudsql-mysql-sa-01"
}
```